packages/f/fx-gltf/xmake.lua

```diff
@@ -0,0 +1,24 @@
+package("fx-gltf")
+    set_homepage("https://github.com/jessey-git/fx-gltf")
```

Comment:
we need set headeronly kind, like this

https://github.com/xmake-io/xmake-repo/blob/c1f4645941199156d5a1836267d4597bd2364acc/packages/d/dr_mp3/xmake.lua#L3

Reply:
ooh i didn't know about this, i was looking for it, thanks!